lib/phoenix_live_view/igniter/upgrade_to_1_1.ex

```diff
@@ -0,0 +1,243 @@
+defmodule Phoenix.LiveView.Igniter.UpgradeTo1_1 do
```

Comment:
We need to wrap this on a Code.ensure_loaded?(Igniter) block.

Reply:
good catch!

Reply:
Another option is to change `elixirc_paths` depending on if igniter is there or not. Then we can keep all igniter things in a igniter/ root folder. If we do this, we need to update the files included the Hex package in the mix.exs.

Reply:
As you prefer!

Reply:
Whatever way works. Your call. :)